packages/grid/src/vaadin-grid-styles.js

```diff
@@ -32,6 +32,13 @@ registerStyles(
       pointer-events: none;
     }
 
+    :host([disable-text-selection]) {
```

Comment:
Would prefer adding the attribute to an element inside the shadow root instead of adding yet another host attribute (which kind of makes it public API). `#scroller[column-resizing]` already has these, could consider reusing (and renaming) it.

Reply:
That would make sense, yes.

Reply:
Done

Reply:
Renaming the attribute has some unfortunate side-effects in that the column resizing logic now has several checks for `disable-text-selection`, which are confusing as it's not apparent how the attribute is related to column resizing. 

From testing this, it seems that the styles are not needed anymore after preventing the default on the mouse down event. I'll rollback the renaming, and update the selection test to check for preventing the mousedown event instead.

Reply:
Done